### DISCUSSION TITLE
HOW WORK IN WEBUI TAB INSTEAD OF NEW PAGE

### DISCUSSION THREAD

**Discussion by BursonHandsom** [ModelSurge/sd-webui-comfyui#67] [Q&A] [answered]
A little confused.

**Comment by PladsElsker** [comment, marked as answer]
Hi, thanks for trying out the extension! 

Right now, in its current state, you should be able to use ComfyUI normally, with the addition of 2 other nodes:
- Load Webui Model
- Webui Save Image

If you want to know how to use ComfyUI itself, I highly suggest that you check on youtube, there are numerous tutorial for how to use it. 
Alternatively, you can check the wiki pages for other extra info on the extension.

**Reply by PladsElsker** [reply]
I transfered the issue into a discussion as I thought it was more appropriate
